README.md

```diff
@@ -31,6 +31,7 @@ Not all versions of StyleCop.Analyzers support all features of each C# language
 |------------|----------------------------|-----------------------|
 |  1.0 - 6.0 | v1.0.2 or higher           | VS2015+               |
 |  7.0 - 7.3 | v1.1.0-beta or higher      | VS2017                |
```

Comment:
💡 Seems this line can be updated at the same time

```suggestion
|  7.0 - 7.3 | v1.1.0-beta or higher      | VS2017+               |
```

Reply:
Done